src/main/resources/view/InternshipListCard.fxml

```diff
@@ -31,6 +31,7 @@
       <Label fx:id="role" styleClass="cell_small_label" text="\$role" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="stage" styleClass="cell_small_label" text="\$stage" />
```

Comment:
personal opinion is that stage should be below role in the UI, open to discussion

Reply:
Let's discuss this as a team.